### PR TITLE
Add explicit generation fields

### PR DESCRIPTION
**Pull request by ProfMoo**
The previous generation relied on `go_package` fields in the `coinbase.staking.rewards` package, which we removed. This change ensures the generation still works.

### CODE REVIEW

File: docs/openapi/orchestration.swagger.json [hunk 1]
```diff
@@ -481,7 +481,23 @@
             "in": "body",
             "required": true,
             "schema": {
-              "$ref": "#/definitions/StakingServicePerformWorkflowStepBody"
```
Comment (ProfMoo) [inline docs/openapi/orchestration.swagger.json:484]: Not sure why this change showed up. Underlying package update is my best guess. I verified that they're functionally the same though.

**Review by marcin-cb — APPROVED** [review]
